views/templates/hook/back-office-header.tpl

```diff
@@ -0,0 +1,12 @@
+{*
+* 2014-2015 Stigmi
+*
+* @author Serge <serge@stigmi.eu>
+* @copyright 2014-2015 Stigmi
+* @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+*}
+<link rel="stylesheet" href="{$module_dir|escape}/views/css/admin-bpost.css" type="text/css" />
```

Comment:
Currently, it breaks the tabs of the 1.4 layout if we do this.
![settings 1 4](https://cloud.githubusercontent.com/assets/7383153/9106720/9eb48f82-3c22-11e5-884a-2acc5f0941c0.png)

We are currently working on the back office for version 1.3, so we'll look into calling these files more selectively. One of the issues we came across is that addJS and addCSS are controller functions >> not available in every hook and part of context.